aiida_common_workflows/workflows/relax/workchain.py

```diff
@@ -63,8 +63,8 @@ def run_workchain(self):
 
     def inspect_workchain(self):
         """Inspect the terminated workchain."""
+        cls = self._process_class.__name__
```

Comment:
Sorry @bosonie , I didn't catch this before, but this problem is also present in the `CommonBandsWorkChain`. Please add the fix there as well